src/ts/documentation.ts

```diff
@@ -10,8 +10,6 @@ export function printDocumentation(
     const printer = new Printer(indentLevel);
     printer.printLn("/**");
     printer.printLn(` * @${type} ${name}`);
-    printer.printLn(" * @author Jonathan Casarrubias <t: johncasarrubias>");
```

Comment:
I just don't think we should add static `@author` on generated files... I think we can have it configurable.